Makefile

```diff
@@ -2,7 +2,7 @@
 
 SHELL:=/bin/bash
 
-all: bootstrap mgmt-deploy build-api-image push-api-image build-resource-processor-vm-porter-image push-resource-processor-vm-porter-image tre-deploy
+all: bootstrap mgmt-deploy build-api-image push-api-image build-resource-processor-vm-porter-image push-resource-processor-vm-porter-image push-gitea-image tre-deploy
```

Comment:
For the other images we have a build and push stage - might use build if want to test locally without push - can we do the same for gitea?

Reply:
Fixed.